hexa/workspaces/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+from django.core.exceptions import ObjectDoesNotExist
+
+from hexa.core.test import TestCase
+from hexa.user_management.models import User
+from hexa.workspaces.models import (
+    Workspace,
+    WorkspaceMembership,
+    WorkspaceMembershipRole,
+)
+
+
+class WorkspaceTest(TestCase):
+    USER_SERENA = None
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.USER_SERENA = User.objects.create_user(
+            "serena@bluesquarehub.com",
+            "serena's password",
+        )
+
+    def test_create_workspace(self):
+        self.assertEqual(0, Workspace.objects.all().count())
+        workspace = Workspace(
+            name="Senegal Workspace", description="This is test for creating workspace"
+        )
+        workspace.save()
+        self.assertEqual(1, Workspace.objects.all().count())
+
+    def test_get_workspace_by_id(self):
+        workspace = Workspace(
+            name="Senegal Workspace", description="This is test for creating workspace"
+        )
+        workspace.save()
+        self.assertEqual(workspace, Workspace.objects.get(id=workspace.id))
+
+    def test_get_workspace_by_id_failed(self):
+        with self.assertRaises(ObjectDoesNotExist):
+            Workspace.objects.get(pk="7bf4c750-f74b-4ed6-b7f7-b23e4cac4e2c")
+
+    def test_add_member(self):
+        workspace = Workspace(
+            name="Senegal Workspace", description="This is test for creating workspace"
+        )
+        workspace.save()
+        workspace_member_ship = WorkspaceMembership.objects.create(
+            workspace=workspace,
+            user=self.USER_SERENA,
+            role=WorkspaceMembershipRole.ADMIN,
+        )
+        workspace_member_ship.save()
+        self.assertEqual(
```

Comment:
I don't think that trying to check if the `id` of the new membership you created is in db is a great test. It's better to test if it's been added to the workspace's memberships

Reply:
Okay.